services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -289,7 +289,7 @@ async def get_file(
         file_id=file_id,
         storage_client=storage_client,
         user_id=user_id,
-        access_right="read",
+        access_right="write",
```

Comment:
Since this small change is so critical, please add here a link to this PR in case someday we are tempted to change the access-rights. At least we are warned and think it twice before doing it :-)

Reply:
As discussed in person I have changed the endpoint in storage to not include `access_rights`.